pipelines/scipipe/ap_verify.groovy

```diff
@@ -281,8 +281,6 @@ def void verifyDataset(Map p) {
               namespace: '',
               datasetName: ds.name,
             )
-            // Delegate upload to Gen 2 code
-          case 2:
```

Comment:
Shouldn't the whole block below this, until `default` also be removed?

Reply:
No, because that code is _also_ needed for Gen 3 (per the deleted comment).

It should be possible to phase out the entire `lsst.verify.Job` class, Job files, etc., but the `faro` group has been resistant to the idea, and none of us want to deal with it during the migration.